OracleWebLogic/samples/wls-k8s-domain/build.sh

```diff
@@ -0,0 +1,2 @@
+
+docker build -t wls-k8s-domain .
```

Comment:
This is not sufficient for a `build.sh` file as it provides no additional functionality over running the build command manually. Please review the other `buildDockerImage.sh` scripts to see what kind of functionality is provided.